quantmsutils/diann/dianncfg.py

```diff
@@ -80,16 +81,22 @@ def convert_mod(unimod_database, fix_mod: str, var_mod: str) -> Tuple[List, List
                 or "iTRAQ" in diann_mod
                 or "mTRAQ" in diann_mod
             ):
-                fix_ptm.append(diann_mod + "," + site + "," + "label")
+                merged[diann_mod].extend(list(site + "," + "label"))
```

Comment:
I think this is wrong.

https://github.com/vdemichev/DiaNN?tab=readme-ov-file#multiplexing-using-plexdia